specs/discussions/2025-05-segments-global-history.md

```diff
@@ -0,0 +1,320 @@
+# Shard segments submission to the beacon chain
+
+## Committing segments to the global history in the beacon chain
+
+The goal of this discussion is to surface the core data structures and mechanics of the process of
+committing segments to the global history in the beacon chain.
+
+1. Child shards create new segments with 128 underlying pieces each with their
+   `record || record_root || parity_record_chunks_root || record_proof`. Each child shard creates
+   `RawSegments` independently (in the same way that is currently implemented). `RawSegments` are
+   assigned and increasing sequence number, `local_index` that determines the order in which they
+   were created in in the shard. These segments that haven't been committed to the global history in
```

Comment:
There are 256 pieces in a segment, but this is again one of those cases where there is too much explanation for things that don't really matter here. `RawSegments` do not currently exist in the code and not used anywhere in the document except in this paragraph, so maybe just say it is done exactly the same way as right now and use the same name as in the code?